Alert/Alert.css

```diff
@@ -0,0 +1,82 @@
+body {
+    font-family: "Roboto", sans-serif;
+    font-size: 0.8rem;
+  }
+  .wrapper {
+    align-items: center;
+    display: flex;
+    flex-direction: column;
+    gap: 2rem;
+    justify-content: center;
+  }
+  .alert-container {
+    align-items: center;
+    background-color: #f0f0f0;
+    border-radius: 0.3rem;
+    display: flex;
+    flex-direction: column;
+    gap: 1rem;
+    justify-content: center;
+    padding: 2rem;
+    width: 600px;
+  }
+  
+  .alert {
+    border-radius: 0.3rem;
+    color: white;
+    overflow-x: auto;
+    padding: 0rem 1rem 0rem 1rem;
+    width: 80%;
+  }
+  
+  .fa {
+    padding-right: 1rem;
+  }
+  
+  .closebtn {
+    color: white;
+    cursor: pointer;
+    float: right;
+    font-size: 22px;
+    font-weight: bold;
+    line-height: 1.2rem;
+    margin-left: 15px;
+    padding-top: 0.8rem;
+    transition: 0.3s;
+  }
+  
+  .closebtnjs {
+    color: white;
+    cursor: pointer;
+    float: right;
+    font-size: 22px;
```

Comment:
anywhere using size above 10px, use rem.

Reply:
Made the changes.